packer/endpointserver.yaml

```diff
@@ -0,0 +1,88 @@
+builders:
+  - type: amazon-ebs
+    ami_name: Packer build for endpointserver 8
+    #get a bit more compute, we are compiling stuff
+    instance_type: t2.large
+    source_ami: ami-c90195b0
+    ssh_username: ec2-user
+#built 1st version at ami-7ca0cc05. need to check how to install memcached
+#up-to-date ami at ami-f39af68a
+provisioners:
+  - type: file
+    source: packer/files/nginx.conf
+    destination: /tmp/nginx.conf
+    
+  - type: file
+    source: packer/files/www.conf
+    destination: /tmp/www.conf
+
+  - type: shell
+    inline:
+      #needed to ensure that awscli actually installs
+      #https://bugzilla.redhat.com/show_bug.cgi?id=1417304
+      - sudo yum-config-manager --enable rhui-REGION-rhel-server-optional
+      - sudo yum -y update
+      - sudo rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+
+      - sudo yum -y install epel-release
+      - sudo yum -y groupinstall 'Development Tools'
```

Comment:
don't think we need this if not building, but working on that right now

Reply:
Possibly not, but may break something if removed, so I say keep it.